pkg/controllers/testing/table.go

```diff
@@ -86,17 +93,18 @@ func (r *Testcase) Test(t *testing.T, scheme *runtime.Scheme, factory Factory) {
 	tracker := createTracker()
 	log := TestLogger(t)
 	c := factory(t, r, clientWrapper, tracker, log)
+
 	// Run the Reconcile we're testing.
-	if _, err := c.Reconcile(reconcile.Request{
+	result, err := c.Reconcile(reconcile.Request{
 		NamespacedName: r.Key,
-	}); (err != nil) != r.ShouldErr {
+	})
+
+	if (err != nil) != r.ShouldErr {
```

Comment:
nit: `r` as a variable name is not very clear

Reply:
`r` -> `tc`